src/coreclr/jit/emitarm64.cpp

```diff
@@ -7884,7 +7884,22 @@ void emitter::emitIns_R_S(instruction ins, emitAttr attr, regNumber reg1, int va
             isSimple = false;
             size     = EA_SCALABLE;
             attr     = size;
-            fmt      = isVectorRegister(reg1) ? IF_SVE_IE_2A : IF_SVE_ID_2A;
+            if (isPredicateRegister(reg1))
+            {
+                assert(offs == 0);
+                // For predicate, generate based of rsGetRsvdReg()
```

Comment:
nit: "off" not "of"

Reply:
I will probably do this in a follow-up PR. Do not want to run CI for this.

Reply:
fixed